.github/workflows/build_trino.yaml

```
@@ -35,3 +35,4 @@ jobs:
       product-name: trino
       sdp-version: ${{ github.ref_type == 'tag' && github.ref_name || '0.0.0-dev' }}
       registry-namespace: sdp
+      runners: ubicloud
```

I think it would be great if you could add a short comment on the motivation of using ubicloud here
```suggestion
      # We have a too high disk usage for GitHub runners
      runners: ubicloud
```